R/calcEDGETransport.R

```diff
@@ -30,6 +30,7 @@ calcEDGETransport <- function(subtype) {
       weight <- as.magpie(weight)
       unit <- "2017US$/(p|t)km"
       description <- "Capital cost (purchase) per energy service demand on CES level."
+      aggregationArguments <- list(zeroWeight = "allow")
```

Comment:
@alexkoberle @johannah-pik 

The subtypes "f35_esCapCost"  and "f34_fe2es"  throw a warning `~ Weight sum is 0, so cannot normalize and will return 0 for some aggregation targets. This changes the total sum of the magpie object! If this is really intended set zeroWeight = "allow", or "setNA" to return NA.`

This happens whenever the chosen weight is zero for one of the aggregation regions - resulting in the values of that region to be zero due to multiplication with 0. Sometimes this is a problem, sometimes it isn't. 

Could you take a look and let me know if this is fine?